### DISCUSSION TITLE
I deployed librenms with docker in pve, but there is nothing to display when I open the web

### DISCUSSION THREAD

**Discussion by be-engineer** [librenms/docker#375] [Q&A]
#### Steps to reproduce this issue

1. open PVE shell
2. mkdir librenms
3. cd librenms
4.  Create 3 files 
root@pve:~/app/librenms# ll
total 20
-rw-r--r--  1 root             root 2961 Apr  6 16:17 docker-compose.yml
-rw-r--r--  1 root             root  291 Apr  6 16:16 librenms.env
-rw-r--r--  1 root             root  204 Apr  6 16:17 msmtpd.env
5.  docker-compose up -d

#### Expected behaviour
open http://ip:8000 should see librenms's login UI.

#### Actual behaviour
http://192.168.152.100:8000 have nothing to display, and no error messages display.
I try to test it using curl, result like this:
root@pve:~/app/librenms# curl http://192.168.152.100:8000 -v
* Expire in 0 ms for 6 (transfer 0x55e9cccd0fb0)
*   Trying 192.168.152.100...
* TCP_NODELAY set
* Expire in 200 ms for 4 (transfer 0x55e9cccd0fb0)
* Connected to 192.168.152.100 (192.168.152.100) port 8000 (#0)
> GET / HTTP/1.1
> Host: 192.168.152.100:8000
> User-Agent: curl/7.64.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Server: nginx
< Date: Tue, 06 Apr 2021 08:33:29 GMT
< Content-Type: text/html; charset=UTF-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Keep-Alive: timeout=10
< Vary: Accept-Encoding
< 
* Connection #0 to host 192.168.152.100 left intact

### Configuration

* Docker version (type `docker --version`) ：Docker version 20.10.5, build 55c4c88
* Docker compose version if applicable (type `docker-compose --version`) : docker-compose version 1.21.0, build unknown
* Platform (Debian 9, Ubuntu 18.04, ...) : 
Distributor ID: Debian
Description:    Debian GNU/Linux 10 (buster)
Release:        10
Codename:       buster
* System info (type `uname -a`) : Linux pve 5.4.44-1-pve #1 SMP PVE 5.4.44-1 (Fri, 12 Jun 2020 08:18:46 +0200) x86_64 GNU/Linux
* Include all necessary configuration files : `docker-compose.yml`, `.env`, ...
Just copied from https://github.com/librenms/docker/tree/master/examples/compose. Only modifyied version: "**3.3**"
### Docker info

```
> Output of command `docker info`
root@pve:~/app/librenms# docker info
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Build with BuildKit (Docker Inc., v0.5.1-docker)

Server:
 Containers: 13
  Running: 7
  Paused: 0
  Stopped: 6
 Images: 7
 Server Version: 20.10.5
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 05f951a3781f4f2c1911b05e61c160e9c30eaa8e
 runc version: 12644e614e25b05da6fd08a38ffa0cfe1903fdec
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.4.44-1-pve
 Operating System: Debian GNU/Linux 10 (buster)
 OSType: linux
 Architecture: x86_64
 CPUs: 3
 Total Memory: 2.861GiB
 Name: pve
 ID: ANFW:FHBG:OTHV:OMY3:2ZFN:D2ZS:JYCH:6LJ3:CU4B:DHDX:NCKE:RI5R
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
```

### Logs

```
> Container logs (set LOG_LEVEL to debug if applicable)
```
NO log.

**Comment by be-engineer** [comment]
Help from Librenms Community.

**Reply by Perkinje** [reply]
At first glance, you're missing the .env file. You should have 4 files.
